plugin/src/main/groovy/com/btkelly/gnag/utils/ProjectHelper.groovy

```diff
@@ -20,18 +20,31 @@ import org.gradle.api.Project
 /**
  * Created by bobbake4 on 4/19/16.
  */
-class ReportHelper {
+class ProjectHelper {
 
-    private final Project project;
+    private final Project project
 
-    ReportHelper(Project project) {
+    ProjectHelper(Project project) {
         this.project = project
     }
 
-    // todo: this doesn't seem like it belongs in the ReportHelper. refactor?
-    public List<File> getAndroidSources() {
-        project.android.sourceSets.inject([]) {
-            dirs, sourceSet -> dirs + sourceSet.java.srcDirs
+    public Project getProject() {
+        return project
+    }
+
+    public boolean isAndroidProject() {
+        return project.hasProperty("android")
```

Comment:
I'm thinking I like the `getExtensions().findByName("android")` a bit better but unless we need the Android dependency I think I would leave it out. If we do end up supporting variants and build types we can refactor.